dashboard/src/modules/components/NavbarDrawerComponent/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import {
+  Masthead,
+  MastheadToggle,
+  PageToggleButton,
+} from "@patternfly/react-core";
+import BarsIcon from "@patternfly/react-icons/dist/js/icons/bars-icon";
+import { useDispatch, useSelector } from "react-redux";
+import { NAVBAR_CLOSE, NAVBAR_OPEN } from "../../../actions/types";
+function NavbarDrawer() {
+  const isNavOpen = useSelector((state) => state.navOpen.isNavOpen);
+  const dispatch = useDispatch();
+  const onNavToggle = () => {
+    const navAction = isNavOpen
+      ? { type: NAVBAR_CLOSE }
+      : { type: NAVBAR_OPEN };
+    dispatch(navAction);
+  };
+  return (
+    <Masthead id="basic">
+      <MastheadToggle>
+        <PageToggleButton isNavOpen={isNavOpen} onNavToggle={onNavToggle}>
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+    </Masthead>
+  );
+}
+
+export default NavbarDrawer;
```

Comment:
Why was this module added to this PR?  (I don't think this addition addresses any outstanding comments, and I think it can be deferred to a subsequent PR...let's not make this PR any bigger if we don't have to!)

Reply:
No,it was a change which was requested!!It might seem that I have added new code(since I created a new module),but I just picked the NavbarDrawer function from the Table component and put it in its separate module since we wanted to create a Common layout for Navigation Drawer and Sidebar (So that we dont want to specify same piece of code in every page of our dashboard,as a Navigation Bar with a drawer will be required in every page).

[The change was requested here](https://github.com/distributed-system-analysis/pbench/pull/2733#discussion_r845977502)

All other changes (except for passing string to constructUTCDate function and timezone splitting one) will be deferred to a new PR

Reply:
My apologies, @shubham-html-css-js.

I'll defer to @MVarshini and @portante as to whether they still think it should be included.